src/sentry/models/monitorenvironment.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+    sane_repr,
+)
+from sentry.models.monitor import MonitorStatus
+
+
+@region_silo_only_model
+class MonitorEnvironment(Model):
+    __include_in_export__ = True
+
+    monitor = FlexibleForeignKey("sentry.Monitor")
+    environment_project = FlexibleForeignKey("sentry.EnvironmentProject")
+    status = BoundedPositiveIntegerField(
+        default=MonitorStatus.ACTIVE, choices=MonitorStatus.as_choices()
+    )
+    next_checkin = models.DateTimeField(null=True)
+    last_checkin = models.DateTimeField(null=True)
+    date_added = models.DateTimeField(default=timezone.now)
+
+    class Meta:
+        app_label = "sentry"
+        db_table = "sentry_monitorenvironment"
+        indexes = [models.Index(fields=["monitor", "environment_project"])]
```

Comment:
Curious what it'll make this index on, I guess it'll be `monitor_id, environmentproject_id` or something?

Wondering if we should be just directly linking to `Environment` here instead of `EnvironmentProject`?

Reply:
Yeah don't want to add the `_id` explicitly.
https://code.djangoproject.com/ticket/30409

Maybe should be `Environment`...`EP` feels safer though considering the `Monitor` is tied to a `Project`

Reply:
And `Environment` can span multiple `Projects`...

Reply:
Usually we just directly link to an environment, I don't see anything that has a relation to `EnvironmentProject` tbh, but a few models like `Rule` have either `environment_id` or a `environment` foreign key. 
This row is already implicitly associated with a project via the monitor.